### DISCUSSION TITLE
Effects of integration on data that does not need to be integrated...

### DISCUSSION THREAD

**Discussion by franziskadenk** [satijalab/seurat#4340] [Q&A]
Dear Satija Lab Team, 
First of all - thank you for providing Seurat - universally loved by all!
I have a rather strange question: we were analysing some data provided by a collaborator that we had assumed were split over separate 10X chips. We therefore integrated them to eliminate any possible batch effects (using Seurat v3). 
Now, we found out that actually, the samples had been run over 4 wells on the same chip, with the experiment carefully batch controlled. And it turns out that without integration, there are some interesting differences. 
Is it at all possible that integration, when used "unnecessarily", can "smooth" out real biological differences? Or is this not a possibility, and the fact that integration eliminated any differences means that there were batch effects - against all best efforts made to avoid them? 
Thank you very much in advance!

**Comment by dmsalsgh97** [comment]
I struggled with the same problem last week, So I'll try to state my own understanding.
I'm not the team of Satija Lab, first.

Seurat integration function can't distinguish 'batch effects(Normally un-Intended difference between sample) and 'Biologically meaningful differences'(Normally Intended difference between sample).
And users can minimize un-wanted over smoothing of integration by manually put 'sample.tree' on IntegrateData() function.

#3487 might be helpful.
